python/ray/experimental/serve/utils/__init__.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from .priority_queue import PriorityQueue
```

Comment:
```suggestion
from ray.experimental.serve.utils.priority_queue import PriorityQueue
```

Reply:
Actually, I take that back. Let's omit this altogether and leave `PriorityQueue` as `priority_queue.PriorityQueue`.

Reply:
So this file can just be empty essentially.

Reply:
changed as you suggested.